components/SectionsColumn.js

```diff
@@ -108,6 +127,16 @@ export const SectionsColumn = ({
     <div className="sections w-80">
       <h3 className="px-1 text-sm font-medium border-b-2 border-transparent text-emerald-500 whitespace-nowrap focus:outline-none">
         {t('section-column-section')}
+        <span className="pl-2 float-right">{t('section-column-click-reset')}</span>
```

Comment:
Can we move the "Reset" span inside the button so it is also clickable?

Reply:
@katherinepeterson  yes , will do this .

Reply:
this is fixed @katherinepeterson